ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/processed/ProcessedLocalizedConfiguration.kt

```diff
@@ -40,10 +40,16 @@ internal data class ProcessedLocalizedConfiguration(
 
                 // Defaults the badge to relative discount if not specified to match
                 // with pre-existing behavior of this feature
-                val offerBadge = offerOverride?.offerBadge?.processVariables()
-                    ?: context.discountRelativeToMostExpensivePerMonth?.let {
+                // but ONLY IF overrides are NOT being used
+                val offerBadge = if (offerOverride != null) {
+                    offerOverride.let { override ->
+                        override.offerBadge?.processVariables()
+                    }
```

Comment:
The `let` seems not needed here? (Caution: coded this on GitHub, did not check in IDE 😇) 
```suggestion
                    offerOverride.offerBadge?.processVariables()
```

Reply:
Since I've been doing the snapshots anyway, I figured I can take this one along: https://github.com/RevenueCat/purchases-android/pull/2222/commits/6f8a622249abae786a844c5c7cc38a94323b1866

Reply:
Thank yyooooouuuu 🫶